services/bugmon-win/launch.sh

```diff
@@ -28,8 +28,8 @@ EOF
 
 retry python -m pip install git+https://github.com/MozillaSecurity/bugmon-tc.git
 
-ARTIFACT_DEST="$USERPROFILE/bugmon-artifacts"
-TC_ARTIFACT_ROOT="project/fuzzing/bugmon"
+ARTIFACT_DEST="./bugmon-artifacts"
+TC_ARTIFACT_ROOT="project\fuzzing\bugmon"
```

Comment:
Are you sure about this? (`\` vs `/`) Even if it needs to be `\`, you probably need to double them or sh will interpret them.

```
$ a="test\fuzz\backup"
$ echo "$a"
test
    uzackup
$ 
```

Reply:
This was actually unintenional.  This path is a TC resource and should always use `/`.